lib/node_modules/@stdlib/stats/base/dists/gamma/entropy/manifest.json

```
@@ -0,0 +1,86 @@
+{
+  "options": {
+    "task": "build",
+    "wasm": false
+  },
+  "fields": [
+    {
+      "field": "src",
+      "resolve": true,
+      "relative": true
+    },
+    {
+      "field": "include",
+      "resolve": true,
+      "relative": true
+    },
+    {
+      "field": "libraries",
+      "resolve": false,
+      "relative": false
+    },
+    {
+      "field": "libpath",
+      "resolve": true,
+      "relative": false
+    }
+  ],
+  "confs": [
+    {
+      "task": "build",
+      "wasm": false,
+      "src": [
+        "./src/main.c"
+      ],
+      "include": [
+        "./include"
+      ],
+      "libraries": [],
+      "libpath": [],
+      "dependencies": [
+        "@stdlib/math/base/napi/binary",
+        "@stdlib/math/base/special/ln",
+        "@stdlib/math/base/special/gammaln",
+        "@stdlib/math/base/special/digamma"
+      ]
+    },
+    {
+      "task": "benchmark",
+      "wasm": false,
+      "src": [
+        "./src/main.c"
+      ],
+      "include": [
+        "./include"
+      ],
+      "libraries": [],
+      "libpath": [],
+      "dependencies": [
+        "@stdlib/math/base/assert/is-nan",
+        "@stdlib/constants/float64/eps",
+        "@stdlib/math/base/special/ln",
+        "@stdlib/math/base/special/gammaln",
+        "@stdlib/math/base/special/digamma"
+      ]
+    },
+    {
+      "task": "examples",
+      "wasm": false,
+      "src": [
+        "./src/main.c"
+      ],
+      "include": [
+        "./include"
+      ],
+      "libraries": [],
+      "libpath": [],
+      "dependencies": [
+        "@stdlib/math/base/assert/is-nan",
+        "@stdlib/constants/float64/eps",
+        "@stdlib/math/base/special/ln",
+        "@stdlib/math/base/special/gammaln",
+        "@stdlib/math/base/special/digamma"
```

We don't use `is-nan` in any of the C files.

```suggestion
        "@stdlib/constants/float64/eps",
        "@stdlib/math/base/special/ln",
        "@stdlib/math/base/special/gammaln",
        "@stdlib/math/base/special/digamma"
      ]
    },
    {
      "task": "examples",
      "wasm": false,
      "src": [
        "./src/main.c"
      ],
      "include": [
        "./include"
      ],
      "libraries": [],
      "libpath": [],
      "dependencies": [
        "@stdlib/constants/float64/eps",
        "@stdlib/math/base/special/ln",
        "@stdlib/math/base/special/gammaln",
        "@stdlib/math/base/special/digamma"
```